package.json

```diff
@@ -105,7 +105,10 @@
       }
     },
     "rules": {
-      "arrow-parens": ["error", "always"],
+      "arrow-parens": [
+        "error",
+        "always"
+      ],
```

Comment:
@sedubois Okay. @mxstbr I didn't get what you wanted to say. Even [master branch file](https://github.com/mxstbr/react-boilerplate/blob/master/package.json#L107) looks identical to this one.